docs/tutorials/getting_started/index.md_template

```diff
@@ -142,6 +142,11 @@ if __name__ == "__main__":
 
 This code starts the UI server, which makes the interface active and functional.
 
+To start the Taipy application, create `main.py` with the content provided earlier, and run the following command in the terminal:
```

Comment:
We try to stick to 100 chars max. Thx,

```suggestion
To start the Taipy application, create `main.py` with the content provided earlier, and run the following command in the terminal:
```

Reply:
Yes, please. That's what my suggestion is. It just adds a line break.

Thank you,